### ISSUE TITLE
Error: Request failed with status code 406

### ISSUE THREAD

**Issue by kodeine** [RemoteMetering/geteventstore-promise#104]
Hello,
I'm getting the following when trying to create a persistent subscription.

```
[Nest] 18174   - 01/12/2021, 7:03:34 PM   [EventStoreBus] Replace EventBus publisher by Eventstore publish
[Nest] 18174   - 01/12/2021, 7:03:34 PM   [EventStore] Connecting to EventStore
[Nest] 18174   - 01/12/2021, 7:03:34 PM   [EventStoreBus] EventStore connected
[Nest] 18174   - 01/12/2021, 7:03:34 PM   [EventStoreBus] Check if persistent subscription "account" on stream $ce-user needs to be created 
[Nest] 18174   - 01/12/2021, 7:03:34 PM   [EventStoreCore] TcpPackageConnection: connected to [{"host":"127.0.0.1","port":1113}, L{"host":"127.0.0.1","port":59559}, 1bfabf94-ece9-4aac-856f-780424fbf286]. +45ms
[Nest] 18174   - 01/12/2021, 7:03:34 PM   [EventStoreCore] TcpConnection: successfully connected to {"host":"127.0.0.1","port":1113} +3ms
{
  status: 406,
  statusText: 'Requested URI is not available in requested format',
  headers: {
    connection: 'close',
    date: 'Wed, 13 Jan 2021 00:03:33 GMT',
    server: 'Kestrel',
    'content-length': '0',
    'access-control-allow-headers': 'Content-Type, X-Requested-With, X-Forwarded-Host, X-Forwarded-Prefix, X-PINGOTHER, Authorization, ES-LongPoll, ES-ExpectedVersion, ES-EventId, ES-EventType, ES-RequireMaster, ES-RequireLeader, ES-HardDelete, ES-ResolveLinkTos',
    'access-control-allow-methods': '',
    'access-control-allow-origin': '*',
    'access-control-expose-headers': 'Location, ES-Position, ES-CurrentVersion'
  },
  config: {
    url: 'http://admin:changeit@localhost:2113/subscriptions/$ce-user/account/info',
    method: 'get',
    headers: {
      Accept: 'application/json, text/plain, */*',
      'User-Agent': 'axios/0.21.1'
    },
    transformRequest: [ [Function: transformRequest] ],
    transformResponse: [ [Function: transformResponse] ],
    timeout: 0,
```

**Comment by mmuller99** [comment]
Hi @kodeine,  would it be possible to please provide some sample code?

**Comment by kodeine** [comment]
Actually this was solved, i was using latest version instead of 5.0.8.
